src/diffusers/loaders.py

```diff
@@ -1301,6 +1434,12 @@ def unload_lora_weights(self):
             else:
                 self.unet.set_default_attn_processor()
 
+            if self.unet.aux_state_dict_populated:
```

Comment:
```suggestion
            if getattr(self.unet, "aux_state_dict_populated", False):
```

Reply:
Done in [172f572](https://github.com/huggingface/diffusers/pull/4147/commits/172f57266daa9f850847b5854b6b91559d43366f)